homeassistant/components/config/entity_registry.py

```diff
@@ -223,32 +225,32 @@ def websocket_remove_entity(hass, connection, msg):
 
 
 @callback
-def _entry_dict(entry):
+def _entry_dict(entry: er.RegistryEntry) -> dict[str, Any]:
     """Convert entry to API format."""
     return {
         "area_id": entry.area_id,
         "config_entry_id": entry.config_entry_id,
         "device_id": entry.device_id,
         "disabled_by": entry.disabled_by,
+        "has_entity_name": entry.has_entity_name,
         "entity_category": entry.entity_category,
         "entity_id": entry.entity_id,
         "hidden_by": entry.hidden_by,
         "icon": entry.icon,
         "name": entry.name,
+        "original_name": entry.original_name,
```

Comment:
This means we're going to send a lot of extra data. I think that we can merge `has_entity_name` and `original_name` to a single field.

Reply:
Talked to Erik, we're not going to make it more compact.